MIP0/mip0.md

```diff
@@ -148,6 +155,8 @@ The MIP Framework serves to empower each off-chain and on-chain Governance parti
 
 8. **Accepted/Rejected:** The MIP is voted on. If it passes, it is officially accepted and is given the `Accepted` status. If not, the MIP is rejected.
 
+9. **Retrospection:** All `Accepted` MIPs besides MIP0 have a Retrospection Date. If no date is listed then the Retrospection Date is implicitly one year after the MIP was `Accepted` for General MIPs and two years for Technical MIPs. The retrospection process is described in MIP0c14.
```

Comment:
If the intention is to exclude MIP0 can you make it more explicit? See comments on the forum